io.github.nemuelw.proton-desktop.yml

```diff
@@ -0,0 +1,51 @@
+app-id: io.github.nemuelw.proton-desktop
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: proton-desktop
+
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=wayland
+  - --socket=x11
+  - --socket=fallback-x11
+
+modules:
+  - name: proton-desktop
+    sources:
+      - type: archive
+        url: https://github.com/nemuelw/proton-desktop/releases/download/v1.0.0/proton-desktop-v1.0.0.tar.gz
+        sha512: 3ecc8bbba183d2fce1d337833d681441ebc650234530e8ccad235c109e015f708486935ce244e9679df5574b8648c9dc1620868ce9d0dbbb72221c3a140b0495
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/bin
+      - cp -r ./_internal /app/bin/_internal
+      - cp ./proton-desktop /app/bin/proton-desktop
```

Comment:
why not build from source?

Reply:
The app may/will be distributed over other channels besides this one, and so it makes sense to just have a centralized build location from where all distribution channels get the already built version.

Reply:
why excluding aarch64 as well?